packages/components/src/segmented-control/stories/index.js

```diff
@@ -46,7 +46,12 @@ export const _default = () => {
 				</SegmentedControl>
 			</Spacer>
 			<Spacer>
-				<SegmentedControl isAdaptiveWidth label={ label } value="long">
+				<SegmentedControl
+					isAdaptiveWidth
+					label={ label }
+					value="long"
+					hideLabelFromVision={ true }
+				>
```

Comment:
I found it a bit hard to understand the Storybook examples without context and without reading the stories sourcecode.

It would be nice if we split the different instances of `SegmentedControl` over different Stories and / or we used knobs to tweak most of the props interactively

This shouldn't be a blocker, but it'd be nice if it happened in a follow-up PR

Reply:
Hey @ntsekouras, I've been AFK for some time — just checking if there's been any progress on this follow-up PR and if I can help in any way!

Reply:
👋 - I was AFK as well 😄 . I'll have a PR for it really soon and ping you there for review!

Reply:
PR ready here: https://github.com/WordPress/gutenberg/pull/34497

Reply:
Awesome, I'll have a look!